plugins/get-all.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: get-all
+spec:
+  version: "v0.0.1"
+  platforms:
+    - uri: https://github.com/corneliusweig/ketall/releases/download/v0.0.1/bundle.tar.gz
+      sha256: 3ec69406ad0b912c4817c37a07a4b3e7866c46dbfaa6d5b3a2db43004374c43f
+      bin: kubectl-get-all
+      files:
+        - from: ./ketall-linux-amd64
+          to: kubectl-get-all
```

Comment:
[no action needed]
FYI: you don't need to rename the binary like this. :) We automatically symlink it as `kubectl-get_all` (note the underscore).

Without the underscore, if you just drop `kubectl-get-all` binary to $PATH, it won't work as it conflicts with builtin `get` IIRC.